examples/user_guide/17-Dashboards.ipynb

```diff
@@ -65,16 +65,16 @@
    "source": [
     "import param\n",
     "import panel as pn\n",
-    "from holoviews.streams import Params\n",
+    "\n",
+    "variables = ['open', 'high', 'low', 'close', 'volume', 'adj_close']\n",
     "\n",
     "class StockExplorer(param.Parameterized):\n",
     "\n",
     "    rolling_window = param.Integer(default=10, bounds=(1, 365))\n",
     "    \n",
     "    symbol = param.ObjectSelector(default='AAPL', objects=stock_symbols)\n",
     "    \n",
-    "    variable = param.ObjectSelector(default='adj_close', objects=[\n",
-    "        'date', 'open', 'high', 'low', 'close', 'volume', 'adj_close'])\n",
+    "    variable = param.ObjectSelector(default='adj_close', objects=variables)\n",
```

Comment:
In the markdown cell below the class definition (not in the diff) it says:

> To generate a set of widgets to control these parameters we can simply supply the explorer.param accessor...

As a general point, nothing about param has been properly introduced so I doubt that means much to the typical reader. Ideally there would be some great param docs to link to here in order to introduce param properly!

Reply:
@jbednar How are param docs coming along? 😄

Reply:
Almost done; just a couple of steps more to do:

1. Start
2. Finish